modules/ROOT/pages/op-tkn-cli-reference.adoc

```diff
@@ -0,0 +1,25 @@
+[id='pipeline-cli-reference']
+= Pipeline CLI reference
+//include::modules/openshift-developer-cli-attributes.adoc[]
+//include::modules/common-attributes.adoc[]
+:context: pipeline-cli-reference
+toc::[]
+
+
+This section lists basic Tekton CLI commands. For a more detailed description, see link:https://github.com/tektoncd/cli[Tekton Pipelines CLI].
+
```

Comment:
I think we have more details now in our docs, do we want to still send the user to the upstream docs?

Reply:
Right now we are listing only the command name along with an example. We are not explaining the various parameters and values for any of the commands. Therefore, in case user wants to know about the commands in detail, they can refer to tkn cli.